Source/ProgressionSystemRuntime/Public/Data/PSWorldSubsystem.h

```diff
@@ -48,9 +41,33 @@ class PROGRESSIONSYSTEMRUNTIME_API UPSWorldSubsystem : public UWorldSubsystem
 	UFUNCTION(BlueprintPure, Category = "C++")
 	FORCEINLINE class UPSHUDComponent* GetProgressionSystemHUDComponent() const { return PSHUDComponentInternal; }
 
-	/** Returns a current save game instance */
+	/** Returns a current progression settings row by name */
```

Comment:
Wrong comment description, you say `Returns a current progression settings row by name` 3 times above many getters, please update all of them in .h and in .cpp above functions

Reply:
whoops. Fixed: https://github.com/h4rdmol/ProgressionSystem/pull/18/commits/9193bc35e64be0beba7c95f20fcfc13808401c8c